docs/operator-manual/config-management-plugins.md

```diff
@@ -34,6 +34,7 @@ metadata:
   # The name of the plugin must be unique within a given Argo CD instance.
   name: my-plugin
 spec:
+  # The version of your plugin. Optional, will be appended to <metadata.name> if set.
```

Comment:
```suggestion
  # The version of your plugin. Optional. If specified, the Application's spec.source.plugin.name field
  # must be <plugin name>-<plugin version>.
```

Reply:
I've adpoted your wording, and ammended the commit.